app/main.py

```diff
@@ -1,6 +1,41 @@
+import os
+import requests
+
+# Constants
+BASE_URL = "http://api.weatherapi.com/v1/current.json"
+DEFAULT_CITY = "Kyiv"
```

Comment:
The task requires the service to get weather information for Paris city by default. Change `DEFAULT_CITY` to 'Paris' to meet the task requirements .